ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceTaskFragment.kt

```diff
@@ -32,16 +32,23 @@ import dagger.hilt.android.AndroidEntryPoint
  */
 @AndroidEntryPoint(AbstractTaskFragment::class)
 class MultipleChoiceTaskFragment : Hilt_MultipleChoiceTaskFragment<MultipleChoiceTaskViewModel>() {
+  private lateinit var binding: MultipleChoiceTaskFragBinding
 
   override fun onCreateTaskView(inflater: LayoutInflater, container: ViewGroup?): TaskView =
     TaskViewFactory.createWithHeader(inflater)
 
   override fun onCreateTaskBody(inflater: LayoutInflater): View {
-    val binding = MultipleChoiceTaskFragBinding.inflate(inflater)
+    binding = MultipleChoiceTaskFragBinding.inflate(inflater)
     setupMultipleChoice(binding.selectOptionList)
     return binding.root
   }
 
+  override fun onSkip() {
+    super.onSkip()
+
+    setupMultipleChoice(binding.selectOptionList)
```

Comment:
Instead of reinitializing the adapter, shouldn't we just update the list in the adapter?

Reply:
There doesn't appear to be an API for that and I think it would be a bit more complicated if we added that API to the two MultipleChoice adapters because then we would either need to cast the RecyclerView adapter to the MultipleChoiceAdapter types, or a perhaps a new MutlipleChoiceAdapter interface. Either way I think it's simpler as is.